sygnal/apnspushkin.py

```diff
@@ -273,7 +273,9 @@ async def _dispatch_request(
             span.set_tag("apns_reason", response.description)
             if (code, response.description) in self.TOKEN_ERRORS:
                 log.info(
-                    "APNs token %s for pushkin %s was rejected: %d %s",
+                    "(Sandbox: %i) Notification request %s with APNs device token %s for pushkin %s was rejected: %d %s",
```

Comment:
Any reason not to print as a string?

```suggestion
                    "(Sandbox: %s) Notification request %s with APNs device token %s for pushkin %s was rejected: %d %s",
```

Reply:
Just that `0/1` would be more concise than `True/False` but doesn't really matter to me!